flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FCU/A320_Neo_FCU.js

```diff
@@ -312,6 +312,9 @@ class A320_Neo_FCU_Speed extends A320_Neo_FCU_Component {
             }
             this.showSelectedSpeed = _showSelectedSpeed;
             SimVar.SetSimVarValue("L:A32NX_FCU_SPD_MANAGED_DASHES", "boolean", this.isManaged && !this.showSelectedSpeed);
+            if (this.currentValue != _value) {
+                SimVar.SetSimVarValue("L:A32NX_AUTOPILOT_SPEED_SELECTED", "number", _value);
+            }
             this.currentValue = _machActive ? _value * 100 : _value;
```

Comment:
Shouldn't the calculation here also be done for the simvar, since we are comparing `this.currentValue` and `_value` above ?

Reply:
You mean for the mach?

I left it out by intention because users might want to have a better readout in the variable (in case of mach you will get 0.78 instead of 78).